bento2seldon/monitoring.py

```diff
@@ -17,40 +18,18 @@ def __init__(self, bento_service: BentoService) -> None:
         self.version = bento_service.version
         self.namespace = config("instrument").get("default_namespace")
 
-        self._exception_counter = self._create_metric(
```

Comment:
Instead of lazily initializing the metrics, couldn't we include the list of extra fields in Monitor constructor? We could make it `[]` by default to avoid breaking any API.

Reply:
[I thought about doing so](https://github.com/bryanoliveira/bento2seldon/commit/ea6fb8f8c353b4f10d90c46dc95872cfbd2c3372), but I was concerned with the `monitor` property overriding that could happen when using mixins. 

For example: a class that requires extra labels for the `_exception_counter` metric mixed with a class that requires extra labels for the `_reward` metric. Only the last one would have a Monitor instance with the required extra labels.